### DISCUSSION TITLE
N3ME closes while loading a map, what to do?

### DISCUSSION THREAD

**Discussion by jwreaper** [ko4life-net/ko#266] [Q&A]
Got N3ME.exe from this repository, tried opening Eslant map provided by Tahsin on Discord.

N3ME closes itself after attempting to open the map .n3m file or any other.

Plus, while attempting to open Moradon map files provided by Tahsin, endless messages are shown about missing object files

Thank you in advance for your guidance

**Comment by stevewgr** [comment]
If you're using the N3ME from the release section, then it is quite outdated. I should probably draft a new release soon after all the refactoring are done.
Also you don't provide any technical info for us to help you. Maybe provide some steps to reproduce with actual files, what version you used. Also build the project from source and run N3ME in debug mode, you'll see better what happens.

About messages of missing object files, yes, that's normal, since they're not perfect and some n3shape files referencing texture, mesh or collision files that don't exists.

**Reply by jwreaper** [reply]
Basically, i had the english version from N3ME editor i got from some KO4Life link marked as "1298 N3ME Editor (Editor)".

-just replaced the old .exe for the one it's provided in Actions tab as a compiled pack of .exes "Build All #25" marked date "2 days ago", so i assumed it's the latest.

-Tried as base object path default N3ME dir with some extra old Object files i added to it + some "ripped" stuff i got yesterday that were shared on Discord.

-Tried also as base object path JPKO Object folder which files are not packed

-I have always get N3ME to work (worked on Mauja for learning purposes), this was the first time i tried to open a downloaded map file. 

-Same thing happened with old .exe and the one i got from Actions tab (program closing while opening Eslant)

-Using W10 x64, in case it somehow matters.

-Endless missiing object messages happen only while trying to open Moradon map, they appear even when files are already in base path

-As seen on shared screenshots, everything is working for the guys on discord so, i don't know what's missing on my PC lol

-I don't know what extra info should i add...

Thanks for your reply
